cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -57,15 +57,16 @@ func (sub CosmosSub) Start(completionEvent *sync.WaitGroup) {
 	time.Sleep(time.Second)
 	client, err := tmClient.New(sub.TmProvider, "/websocket")
 	if err != nil {
-		sub.Logger.Error("failed to initialize a client", "err", err)
+		sub.SugaredLogger.Errorw("failed to initialize a sifchain client.",
+			"error message", err.Error())
```

Comment:
We should extract out the "error message" and make it a constant so every time we call the logger we have consistent field names.  (And I think we shouldn't put spaces in these keys)

Reply:
accepted.